### PR TITLE
Create journal-week-1.md

### PR DESCRIPTION
**Pull request by ericquayson**

### CODE REVIEW

**Comment by ericquayson** [comment]
https://github.com/ericquayson/ericquayson.github.io/pull/5

**Comment by ericquayson** [comment]
https://github.com/ericquayson/TIY-Assignments/pull/5

File: NestedAccordion/index.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"> 
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+		<title>Blueprint: Nested Accordion</title>
+		<meta name="description" content="Blueprint: Nested Accordion" />
+		<meta name="keywords" content="nested, accordion, simple, vertical, web development, css3, javascript" />
+		<meta name="author" content="Codrops" />
+		<link rel="shortcut icon" href="../favicon.ico">
+		<link rel="stylesheet" type="text/css" href="css/default.css" />
+		<link rel="stylesheet" type="text/css" href="css/component.css" />
+		<script src="js/modernizr.custom.js"></script>
```
Comment (kellymurray) [inline NestedAccordion/index.html:14]: No link to your `scripts.js`?

File: NestedAccordion/index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+										<p>Cookie ice cream sweet I love cupcake. Fruitcake topping wafer. Lollipop liquorice I love tart wypas biscuit. Danish jelly-o gingerbread. Faworki jelly pastry. Wypas wypas topping I love I love candy canes liquorice. I love sweet roll fruitcake dragée wafer icing bonbon.</p>
+									</div>
+								</li>
+							</ul>
+						</div>
+					</li>
+				</ul>
+			</div>
+		</div>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```
Comment (kellymurray) [inline NestedAccordion/index.html:181]: need to remove the JS

**Pull request merged**